redis/client_test.go

```diff
@@ -67,6 +68,11 @@ func TestParse(t *T) {
 	assert.Equal(t, ErrorReply, r.Type)
 	assert.Equal(t, "ERR unknown command 'foobar'", r.Err.Error())
 
+	// wrongtype reply
+	r = parseString("-WRONGTYPE Operation against a key holding the wrong kind of value\r\n")
+	assert.Equal(t, ErrorReply, r.Type)
+	assert.Equal(t, "WRONGTYPE Operation against a key holding the wrong kind of value", r.Err.Error())
+
```

Comment:
Looks good.. what prompted this extra test?
I was looking around for `-WRONGTYPE` and saw it mentioned in 2.8:
https://raw.githubusercontent.com/antirez/redis/2.8/00-RELEASENOTES
>Many errors are now prefixed by a more specific error code instead of the generic -ERR, for example -WRONGTYPE, -NOAUTH, ...